hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -66,7 +66,7 @@ def __init__(self) -> None:
         self.trace: List[TraceDmemStore] = []
         self.pending: Dict[int, int] = {}
 
-    def _load_5byte_le_words(self, data: bytes) -> None:
+    def _load_5byte_le_words(self, data: bytes, offset: int = 0) -> None:
```

Comment:
This looks sensible, but could you extend the documentation comment to mention the new argument? Maybe something as simple as "The loaded bytes start at address offset" or similar.

Similarly for the 4byte and general versions below.